runtime/src/dispatcher.rs

```diff
@@ -170,10 +170,13 @@ pub struct Dispatcher {
 impl Dispatcher {
     #[cfg(target_env = "sgx")]
     fn new_tokio_runtime() -> tokio::runtime::Runtime {
```

Comment:
In the future we could move this to `future.rs`, call it from `start_runtime` and pass it to dispatcher and protocol constructors.

Reply:
Oh, that's a good idea, yeah this part needs some refactoring.